src/utils/pgwire/src/pg_extended.rs

```diff
@@ -0,0 +1,108 @@
+use bytes::{Bytes, BytesMut};
+use regex::Regex;
+
+use crate::pg_field_descriptor::{PgFieldDescriptor, TypeOid};
+
+// NOTE : Code dundancy , may need to modify later
+fn cstr_to_str(b: &Bytes) -> &str {
+    let without_null = if b.last() == Some(&0) {
+        &b[..b.len() - 1]
+    } else {
+        &b[..]
+    };
+    std::str::from_utf8(without_null).unwrap()
+}
```

Comment:
I think we already have this function before

Reply:
yes. but if I use it, pg_extended.rs will rely on the 'cstr_to_str' in pg_protocol.rs.  Should we consider putting 'cstr_to_str' somewhere else, such as lib.rs?

Reply:
> pg_extended.rs will rely on the 'cstr_to_str' in pg_protocol.rs

I think it is acceptable. Or maybe we can put it in util.rs in later refactor PR.